### ISSUE TITLE
Error Message - "Could not import SDD file."

### ISSUE THREAD

**Issue by eomere** [NREL/OpenStudioApplication#81]
Issue overview
--------------

The Import SDD feature fails. 

## Steps to Reproduce
1. Create an SDD file. To do this, I exported an SDD from an OSM using the OS SketchUp Plugin. 
2. Launch the OS App. Follow the menu options: "**File/Import/SDD**".

**Comment by jmarrec** [comment]
Could you share the SDD file so I get a chance to reproduce?

Also, the SDD import itself lives on NREL/OpenStudio so eventually the fix (and issue) would be there.

**Comment by eomere** [comment]
Here is a sample [SDD Model](https://www.dropbox.com/s/ke2gpweqdbj71wm/SDD_SampleModel.xml?dl=0).

**Comment by jmarrec** [comment]
Ok I see. It's confusing... @eomere you're hitting this: https://github.com/jmarrec/OpenStudio/blob/2700a16d28ce7f8bf10cbdf16bfd1b1ac4263b31/src/sdd/ReverseTranslator.cpp#L247-L255

I think This file is supposed to be run by CBECC-Com first, and then you can reload in OSApp the result after simulation.

**Comment by jmarrec** [comment]
Ideally the reporting back to user in OS App would be informative and include the logger message:

![image](https://user-images.githubusercontent.com/5479063/77334909-b4a2b100-6d25-11ea-8746-5a647db2c0b2.png)
Whereas in the CLI I see something useful:

```
[openstudio.sdd.ReverseTranslator] <1> Import of Input SDD XML type is not currently supported
```

**Comment by eomere** [comment]
So you're saying this feature isn't actually supported yet? Does this mean that the feature could be dropped from the App?

**Comment by jmarrec** [comment]
That's not what I'm saying no.

There are two types of SDD XML files: Input, and Simulation files.

Import SDD file works for the Simulation files, not the input ones.

I never used the feature myself but as I understand it:

Sketchup > Export to SDD > Import SDD into CBECC-Com > Run CBECC > Take **simulation** XML file > Import SDD **simulation** Xml in OS App to have an OSM.

**Comment by jmarrec** [comment]
![image](https://user-images.githubusercontent.com/5479063/77339136-a5266680-6d2b-11ea-8a63-38c3658d40b0.png)

**Comment by jmarrec** [comment]
@eomere Give the above, would the fix in #88 suffice to close this one?

**Comment by eomere** [comment]
Yes, thanks for this. Updating the SDD error message so that a User can better understand how the feature works will help.
